### PR TITLE
changes

### PR DESCRIPTION
**Pull request by abuhurairalakdawala**

### CODE REVIEW

File: data/DoctrineORMModule/Proxy/__CG__UserEntityUser.php [hunk 1]
```diff
@@ -0,0 +1,271 @@
+<?php
```
Comment (santoshborate1) [inline data/DoctrineORMModule/Proxy/__CG__UserEntityUser.php:1]: We never commit proxy in github

File: module/User/src/Entity/Post.php [hunk 2]
```diff
@@ -79,4 +85,22 @@ public function getDescription(): string
...
     }
+
+    /**
+     * @param User $user
+     * @return void
+     */
+    public function setUser(User $user): void
+    {
+        $this->user = $user;
+        $user->addPost($this);
```
Comment (santoshborate1) [inline module/User/src/Entity/Post.php:96]: Why we need addPost in method. 
Its avoiding SOLID principle. single responsibility of method.
Here now it set user and and add post also which is not good. 
Also some where else if we need to add post then there also we will duplicate the code

Comment (abuhurairalakdawala) [reply module/User/src/Entity/Post.php]: removed

File: data/DoctrineORMModule/Proxy/__CG__UserEntityUser.php (hunk 1 above)
Comment (abuhurairalakdawala) [reply data/DoctrineORMModule/Proxy/__CG__UserEntityUser.php:1]: added in gitignore

File: module/User/src/Controller/UserController.php [hunk 3]
```diff
@@ -144,4 +154,68 @@ public function deleteAction()
...
+                    'post' => $post,
+                    'postForm' => $postForm
+                ]
+            );
+        }
+
+        $postForm->setData($request->getPost());
+
+        if ($postForm->isValid()) {
+            $post = new Post();
```
Comment (santoshborate1) [inline module/User/src/Controller/UserController.php:194]: Can you please check how we can move this to service to prepare post object

File: module/User/src/Controller/UserController.php [hunk 4]
```diff
@@ -144,4 +154,68 @@ public function deleteAction()
...
+    /**
+     * @return ViewModel
+     * @throws \Doctrine\ORM\ORMException
+     * @throws \Doctrine\ORM\OptimisticLockException
+     */
+    public function editAction(): ViewModel
+    {
+        $id = $this->params()->fromRoute('id');
+        $postForm = new PostForm();
+        $inputFilter = $postForm->getInputFilter();
```
Comment (santoshborate1) [inline module/User/src/Controller/UserController.php:167]: Can you check how we can move input filter form form

**Pull request merged**